io.github.janbar.noson.json

```diff
@@ -0,0 +1,58 @@
+{
+  "app-id": "io.github.janbar.noson",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.10",
```

Comment:
Did you try 5.11?

Reply:
5.11 hasn't been shipped yet, has it?

Reply:
It's available to install from Flathub

Reply:
Doesn't build against 5.11:
> CMake Error at gui/CMakeLists.txt:42 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".

Reply:
> Doesn't build against 5.11:

Fixing this seems to simple: https://phabricator.kde.org/D12272